padle/menu/padle_scroll.py

```diff
@@ -53,8 +53,8 @@ async def respond_with_right(cls, message: Optional[Message], ims, **data):
     async def respond_with_pane(cls, message: Optional[Message], ims, **data) -> EmbedWrapper:
         dbcog = data['dbcog']
         user_config = data['user_config']
-        today_guesses = data['today_guesses']
-        view_state = await PADleScrollViewState.deserialize(dbcog, user_config, today_guesses, ims)
+        padle_cog = data['padle_cog']
+        view_state = await PADleScrollViewState.deserialize(dbcog, user_config, padle_cog, ims)
```

Comment:
nit: change the order here so we're sending the two cogs first - that's a cleaner order of arguments